PENDING.md

```diff
@@ -32,8 +32,14 @@ FEATURES
 IMPROVEMENTS
 
 * Gaia REST API (`gaiacli advanced rest-server`)
+  * \#3091 Update deposit and vote endpoints to perform a direct txs query
```

Comment:
Shouldn’t we use the issue number instead of the PR?

Reply:
@fedekunze I always prefer the PR # over the issue, but we don't have a guideline.

Reply:
+1 on mentioning the issue, if there's any open

Reply:
I am outvoted here lol. Changed.